mmpose/core/post_processing/post_transforms.py

```diff
@@ -112,11 +121,20 @@ def transform_preds(coords, center, scale, output_size):
     assert len(center) == 2
     assert len(scale) == 2
     assert len(output_size) == 2
-
-    target_coords = coords.copy()
-    trans = get_affine_transform(center, scale, 0, output_size, inv=True)
-    for p in range(coords.shape[0]):
-        target_coords[p, 0:2] = affine_transform(coords[p, 0:2], trans)
+    if use_udp:
+        scale = scale * 200.0
```

Comment:
add a comment on the magic number 200

Reply:
'inverse the effect of scale normalize'?
'the scale is normalize by factor of 200.0 in 'mmpose/datasets/datasets/top_down/topdown_coco_dataset.py,line217-244''